basics.rst

```diff
@@ -0,0 +1,59 @@
+Basics
+======
+
+
+Thin Client
+-----------
+
+Electrum's focus is security, recoverability, versatiliy, and speed.
+The servers do the heavy lifting. (without compromising security)
+Your private keys are never shared with the server, only addresses.
+
+
+Seed Words - KEEP THESE SAFE !!!
+----------
+
+Your seed words are used to recover your wallet.   They generate your private keys and addresses.
+
+Example:
+
+.. code-block:: none
+
+   #DO NOT SHARE!
+   tango execute day arena female win object expect final tank kingdom web
+
+
+Address
+-------
+
+Each seed / wallet will generate new addresses for each new transaction.
```

Comment:
Some people might ask whether addresses can be used multiple times. You could rephrase that so to suggest to use a different address for each transaction (for privacy reasons).